src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Extras/MarkdownViewer/BitMarkdownViewerDemo.razor.scss

```
@@ -1,7 +1,69 @@
+@import '../../../../Styles/abstracts/_bit-css-variables.scss';
```

Stylelint (17.13.0)

[error] Unexpected leading underscore in imported partial name (scss/load-no-partial-leading-underscore)

(scss/load-no-partial-leading-underscore)